cmd/livepeer/starter/starter.go

```diff
@@ -54,9 +54,9 @@ var (
 	txCostMultiplier = 100
 
 	// The interval at which to clean up cached max float values for PM senders and balances per stream
-	cleanupInterval = 1 * time.Minute
+	cleanupInterval = 10 * time.Minute
 	// The time to live for cached max float values for PM senders (else they will be cleaned up) in seconds
-	smTTL = 60 // 1 minute
+	smTTL = 604800 // 1 week
```

Comment:
How about setting it to something like 2 days and refreshing the values every round or every 1 day?

Reply:
The SenderMonitor sets a `lastAccess` which is used for clearing the cache, so I agree that the timeout doesn't have to be too high. Retaining B info for as long as they've sent a GetOrchestrator request in the past 2 days makes sense to me

Reply:
I agree, the B sends GetOrchestrator requests at startup and every hour so as long as the B is up the cache should not clear on the Os.